lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -225,6 +227,17 @@ def get_can_delete(self, obj):
         """
         return can_delete(obj, self.context)
 
+    def get_last_edit(self, obj):
+        """
+        Returns information about the last edit for this content for
+        privileged users.
+        """
+        if _validate_privileged_access(self.context):
+            try:
+                return obj["edit_history"][-1]
+            except IndexError:
+                return None
```

Comment:
We also need to check for `KeyError` here, or change this to `obj.get` since it's possible for the API to not return an edit history. Maybe your PR fixes this, but just pointing it out in case it didn't come up for you yet.